tests/eas/load_tracking.py

```diff
@@ -740,6 +740,77 @@ def _test_util_per_cpu(self, experiment, tasks):
                                        msg=msg.format(util_mean[i], cpus[i],
                                                       expected[i], phases[j]))
 
+class OneTaskCPUMigrationTest(_CPUMigrationBase):
+    """
+    Goal
+    ====
+    Checks that the utilization is correctly updated for CPU when a task is
+    migrated from a CPU to another.
+
+    Detailed Description
+    ====================
+    Three tasks are created. Each task defined two phases as follow (the CPU
+    is the one corresponding for the hikey960 it may be another CPU on
+    another platform):
+
+              Phase 0         Phase 1
+            CPU | Util      CPU | Util
+    Task 0:   4 | 25%         5 | 25%
+    Task 1:   4 | 10%         4 | 10%
+    Task 2:   5 | 5%          5 | 5%
+
+    Expected Behaviour
+    ==================
+    The utilization of a CPU is sampled during both phases and should be
+    observed as follow:
+
+            Phase 0         Phase 1
+            Util            Util
+    CPU 4:   35%             10%
+    CPU 5:   5%              30%
+    """
+    @classmethod
+    def _getExperimentsConf(cls, test_env):
+        # Get the 2 CPUs with the highest capacities
+        cap_cpu = sorted(test_env.calibration(),
```

Comment:
Why that name? What about just `cpus`... the meaning is already in the comment.

Reply:
Done